<script src="js/jquery.maphilight.js"></script>
<script src="js/area-popover.js"></script>

<style>
.container{
    min-width: 860px;
}
#workflow .popover{
    display: none;
    position: relative;
}
</style>

<div class="col-xs-8 main-left" style="min-width:555px;">
    <div class="panel panel-default description">
        <div class="panel-heading"><h4>Home <span class="glyphicon glyphicon-paperclip"></span></h4></div>
        <div class="panel-body">
            <h3 class="subTitleFont">Overview</h3>
            <p><strong>D</strong>e Novo <strong>A</strong>ML patient <strong>P</strong>rognostic <strong>P</strong>rediction (<strong>DAPP</strong>) is developed for biologists and clinical researchers to predict the prognosis of de novo Acute Myeloid Leukemia (AML) using machine learning models. DAPP supports both gene mutation and expression data as input.</p>
            <p>We searched published AML data sets with survival information. Only keep non-redundant de novo AML samples with survival information, a total of 3,315 samples were retained, including 1,668 gene expression samples and 1,647 mutation samples. Then, we employed nine machine learning methods to build multiple models. Final, we adjust the scores with a smooth monotonic fit to keep the survival probability score decreases monotonously with time after the diagnosis.</p>
            <hr />
            <!--            <h3 class="subTitleFont">SiteMap</h3>
            <img id="workflow" usemap="#image-map" src="images/fig1.jpg" class="center-block" >
            <map name="image-map" id="workflow_map">
                <area class="areaClass" data-placement="right" target="_blank" alt="" title="" href="?idx=Predict" coords="506,516,65,387" shape="rect" data-toggle="popover" data-trigger="hover" data-content="Go to Predict">
                <area class="areaClass" data-placement="top" target="_blank" alt="" title="" href="?idx=Train" coords="509,306,394,136" shape="rect" data-toggle="popover"  data-trigger="hover" data-content="Go to Train">
                <area class="areaClass" data-placement="right" target="_blank" alt="" title="" href="?idx=Download" coords="19,17,318,238" shape="rect" data-toggle="popover" data-trigger="hover" data-content="Go to Download">
                <area class="areaClass" data-placement="right" target="_blank" alt="" title="" href="?idx=Document#3. Machine Learning Methods" coords="86,315,198,371" shape="rect" data-toggle="popover" data-trigger="hover" data-content="Go to ModelsDoc">
            </map> -->
        </div>
    </div>
    <div class="panel panel-default description">
        <div class="panel-heading"><h4>Gene Expression Datasets <span class="glyphicon glyphicon-paperclip"></span></h4></div>
        <div class="panel-body">
            <h3 class="subTitleFont">Gene Expression Datasets Description</h3>
            <hr />
            <h3 class="subTitleFont">Microarray data</h3>
            <h4>GSE37642</h4>
            <p>The series matrix files of GSE37642 were downloaded from NCBI Gene Expression Omnibus (GEO). 9 samples with missing survival information were removed, and 553 cases were retained.</p>
            <h4>GSE6891</h4>
            <p>The raw CELL files for GSE6891 were downloaded from NCBI Gene Expression Omnibus (GEO). The overall survival data was downloaded from cytopred's github repository https://github.com/narenschandran/cytopred (Salunkhe, et al., 2018). Samples with missing age or survival information were removed, and 505 cases were retained.</p>
            <hr />
            <h3 class="subTitleFont">RNA-seq data</h3>
            <h4>GSE106291</h4>
            <p>The raw reads count files for GSE106291 were downloaded from NCBI Gene Expression Omnibus (GEO). Then calculate the RPKM values of all genes in all samples using the R/Bioconductor package edgeR version 3.24.3.</p>
            <h4>TCGA & BEAT</h4>
            <p>The mRNA expression Z-Scores file of BEAT AML dataset and the mRNA gene expression file of TCGA AML dataset were downloaded from cbioportal (http://www.cbioportal.org/).</p>
            <p>Only the de novo AML patients with overall survival information were kept for this study. GSE106291(n=250), BEAT(n=187) and TCGA(n=173) were retained.</p>
            <hr />
            <h3 class="subTitleFont">Gene Expression Datasets list</h3>
            <table class="table table-sm table-bordered pfont"><tr class="th"><td><span class="text-truncate">Data Set</span></td><td><span class="text-truncate">Data Type</span></td><td><span class="text-truncate">Number of samples</span></td></tr><tr class=""><td><span class="text-truncate">TCGA</span></td><td><span class="text-truncate">RNA</span></td><td><span class="text-truncate">173</span></td></tr><tr class=""><td><span class="text-truncate">BEAT</span></td><td><span class="text-truncate">RNA</span></td><td><span class="text-truncate">187</span></td></tr><tr class=""><td><span class="text-truncate">GSE6891_GPL570</span></td><td><span class="text-truncate">RNA</span></td><td><span class="text-truncate">505</span></td></tr><tr class=""><td><span class="text-truncate">GSE106291_rnaseq</span></td><td><span class="text-truncate">RNA</span></td><td><span class="text-truncate">250</span></td></tr><tr class=""><td><span class="text-truncate">GSE37642_GPL96</span></td><td><span class="text-truncate">RNA</span></td><td><span class="text-truncate">417</span></td></tr><tr class=""><td><span class="text-truncate">GSE37642_GPL570</span></td><td><span class="text-truncate">RNA</span></td><td><span class="text-truncate">136</span></td></tr></table>
        </div>
    </div>
    <div class="panel panel-default description">
        <div class="panel-heading"><h4>Mutation Datasets <span class="glyphicon glyphicon-paperclip"></span></h4></div>
        <div class="panel-body">
            <h3 class="subTitleFont">Mutation Datasets Description</h3>
            <p>The three datasets were used for the Mutation models.</p>
            <p>The data of AMLSG was downloaded from AML-multistage's github repository http://www.github.com/gerstung-lab/aml-multistage (Gerstung, et al., 2017). The mutations data of BEAT AML and TCGA AML dataset were downloaded from cbioportal (http://www.cbioportal.org/). Only the de novo AML patients with overall survival information were kept for this study. AMLSG(n=1234), BEAT(n=217) and TCGA(n=196) were retained.</p>
            <h3 class="subTitleFont">Mutation Datasets list</h3>
            <table class="table table-sm text-truncate pfont"><tr class="th"><td><span class="text-truncate">Data Set</span></td><td><span class="text-truncate">Data Type</span></td><td><span class="text-truncate">Number of samples</span></td></tr><tr class=""><td><span class="text-truncate">AMLSG</span></td><td><span class="text-truncate">MUT</span></td><td><span class="text-truncate">1234</span></td></tr><tr class=""><td><span class="text-truncate">TCGA</span></td><td><span class="text-truncate">MUT</span></td><td><span class="text-truncate">196</span></td></tr><tr class=""><td><span class="text-truncate">BEAT</span></td><td><span class="text-truncate">MUT</span></td><td><span class="text-truncate">217</span></td></tr></table>
        </div>
    </div>
</div>

<div class="col-xs-4 main-right">
    <div class="panel panel-default description">
        <div class="panel-heading"><h4>Cite <span class="glyphicon glyphicon-paperclip"></span></h4></div>
        <div class="panel-body">
            <p>High Temporal Resolution of Mortality Risk Identification for single AML Patient via Deep Learning-Based Prediction (unpublished).</p>
        </div>
    </div>
</div>
